Jenkinsfile

```diff
@@ -43,11 +43,7 @@ pipeline {
         }
         stage('Analyze') {
             steps {
-                sh 'mvn apache-rat:check'
-                sh 'mvn clirr:check'
-                sh 'mvn checkstyle:check'
-                sh 'mvn spotbugs:check'
-                sh 'mvn javadoc:javadoc'
+                sh 'mvn apache-rat:check clirr:check checkstyle:check spotbugs:check javadoc:javadoc'
```

Comment:
What about just calling `mvn` which does pretty much the same, see: https://github.com/apache/commons-lang/blob/master/pom.xml#L627 ?

Reply:
Might be a good idea. I think in Travis we have that in some other components too... that way once updated, we won't have to touch the Jenkinsfile (which I'm less familiar than pom.xml). Or would there be some advantage in maintaining the build maven goals in the Jenkinsfile instead?

Reply:
Okay, let's do it like we do it for travis. In this case I would also combine the build and analyze stages into one stage.

Reply:
@kinow please take another look. Can we merge the pipeline script like this?

Reply:
+1 LGTM